src/components/Composer.vue

```diff
@@ -653,6 +653,18 @@ export default {
 					})
 					this.state = STATES.ERROR
 				})
+
+			// Sync sent mailbox when it's currently open
+			const account = this.$store.getters.getAccount(this.selectedAlias.id)
+			if (this.$route.params.mailboxId === account.sentMailboxId) {
```

Comment:
isn't ``this.$route.params.mailboxId`` a `string` and the `sentMailboxId` a `Number`? I think we have to use `parseInt(this.$route.params.mailboxId, 10) === account.sentMailboxId`